mysql-test/main/type_ranges.result

```diff
@@ -144,8 +144,10 @@ alter short drop default,
 DROP INDEX utiny,
 DROP INDEX ushort,
 DROP PRIMARY KEY,
-DROP FOREIGN KEY any_name,
+DROP FOREIGN KEY IF EXISTS any_name,
 ADD INDEX (auto);
+Warnings:
+Note	1091	Can't DROP FOREIGN KEY `any_name`; check that it exists
```

Comment:
The "Warnings" for the "Note" seems a bit inappropriate here, but this logic was not changed by this patch.

Just out of curiosity: What would `ALTER TABLE t1 DROP COLUMN IF EXISTS non_existent_column` report?

Reply:
ALTER TABLE t2 DROP FOREIGN KEY IF EXISTS fk_1;
Warnings:
Note	1091	Can't DROP FOREIGN KEY `fk_1`; check that it exists